gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Jan 12 19:57:42 EST 2017
+#Mon May 01 15:13:13 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-all.zip
```

Comment:
This file gets updated like this accidentally a lot.  When it's generated it's created with `bin`, but Intellij always suggest changing it to `all` so people do that and then accidentally commit it.  It doesn't really matter either way, if it's set to bin then it only downloads the gradle binaries, with all it also downloads the source jars so it can be inspected locally more easily.